Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -169,9 +169,10 @@ union ShaderHostConfig
   BitField<20, 1, bool, u32> backend_shader_framebuffer_fetch;
   BitField<21, 1, bool, u32> backend_logic_op;
   BitField<22, 1, bool, u32> backend_palette_conversion;
-  BitField<23, 1, bool, u32> enable_validation_layer;
-  BitField<24, 1, bool, u32> manual_texture_sampling;
-  BitField<25, 1, bool, u32> manual_texture_sampling_custom_texture_sizes;
+  BitField<23, 1, bool, u32> backend_sampler_lod_bias;
+  BitField<24, 1, bool, u32> enable_validation_layer;
+  BitField<25, 1, bool, u32> manual_texture_sampling;
+  BitField<26, 1, bool, u32> manual_texture_sampling_custom_texture_sizes;
```

Comment:
The order of these fields is important (they're reflected in the shader cache filenames, e.g. `D3D-specialized-pipeline-00000000-61FF40.cache`/`D3D-specialized-pipeline-00000000-61FF60.cache`/`D3D-specialized-pipeline-00000000-E1FF40.cache`).  If you add a new field, you should put it at the end.

Reply:
Fixed, thanks... Thought this order didn't matter and tried to keep backend stuff together